src/main/java/seedu/address/logic/parser/cookbook/CookbookAddCommandParser.java

```diff
@@ -52,14 +52,17 @@ public CookbookAddCommand parse(String args) throws ParseException {
         }
 
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_RECIPE_NAME, PREFIX_RECIPE_DESCRIPTION, PREFIX_INGREDIENT_NAME,
-                    PREFIX_INGREDIENT_QUANTITY, PREFIX_STEP_INDEX, PREFIX_STEP_DESCRIPTION, PREFIX_TAG);
+            ArgumentTokenizer.tokenize(args, PREFIX_RECIPE_NAME, PREFIX_RECIPE_DESCRIPTION, PREFIX_INGREDIENT_NAME,
+                PREFIX_INGREDIENT_QUANTITY, PREFIX_STEP_INDEX, PREFIX_STEP_DESCRIPTION, PREFIX_TAG);
```

Comment:
According to the 2103 java coding style, we should have 8 spaces indentation for line wrapping, though the checkstyle script does not check for this,